src/main/java/com/dev/cinema/model/Order.java

```diff
@@ -0,0 +1,65 @@
+package com.dev.cinema.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    private LocalDateTime orderDate;
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
No need to set the fetch type **_LAZY_** in the connections `@ManyToOne` since you fetch the user in your query
Let's delete this part in the brackets and fetching in your DAO layer method

Reply:
thanks